screwdriver/screwdriver.go

```diff
@@ -73,7 +73,8 @@ func New(url, token string) (API, error) {
 
 // BuildStatusPayload is a Screwdriver Build Status payload.
 type BuildStatusPayload struct {
-	Status string `json:"status"`
+	Status string                 `json:"status"`
+	Meta   map[string]interface{} `json:"meta"`
```

Comment:
`meta` is defined in data-scheme: 
https://github.com/screwdriver-cd/data-schema/blob/master/models/build.js#L151